pkg/services/build.go

```diff
@@ -13,8 +14,8 @@ import (
 
 type IBuildService interface {
 	Create(CreateBuildDTO) (string, error)
-	Find(filter *BuildFilter) (*BuildDTO, error)
-	List(filter *BuildFilter) ([]*BuildDTO, error)
+	Find(filter *BuildFilter, params BuildRetrievalParams) (*BuildDTO, error)
```

Comment:
Why is this a separate type and argument?

`ShowDeleted` can be a filter option as well

Reply:
true